pkg/kubelet/kubelet_server_journal_windows.go

```diff
@@ -68,27 +68,13 @@ func getLoggingCmdArgs(includeSinceTime, includeUntilTime, includeTailLines, inc
 		"-Command",
 	}
 
-<<<<<<< HEAD
-	psCmd := "Get-WinEvent -FilterHashtable @{LogName='Application'"
-	if len(n.Since) > 0 {
-		psCmd += fmt.Sprintf("; StartTime='%s'", n.Since)
-	} else if n.SinceTime != nil {
-		psCmd += fmt.Sprintf("; StartTime='%s'", n.SinceTime.Format(dateLayout))
-	}
-
-	if len(n.Until) > 0 {
-		psCmd += fmt.Sprintf("; EndTime='%s'", n.Until)
-	} else if n.UntilTime != nil {
-		psCmd += fmt.Sprintf("; EndTime='%s'", n.UntilTime.Format(dateLayout))
-=======
```

Comment:
This is also part of https://github.com/openshift/kubernetes/commit/d3441b4a3b5a and we can't remove it.

Reply:
Ok, I've pushed a PR that takes that side instead, but it doesn't work because the function signature of getLoggingCmdArgs changed (in 9c25992dcbe5af56e032ea68f9e93fa55c8a68d3). How do you suggest proceeding?